client/js/modalContent.js

```diff
@@ -0,0 +1,78 @@
+const getTitleAndDateFromModal = async() => {
+    /** 
+     *Funtion to read the title and data from the modal DOM and inject new rating div if one doesn't exist
+     * @param {}
+     * @return {}
+    */
+
+    // Reads the title name of movie / series from the previewModal--player-titleTreatment-logo class
+    let titleElement = document.querySelectorAll('.previewModal--player-titleTreatment-logo')
+    let title = (titleElement === undefined  || titleElement[0] === undefined ) ? null : titleElement[0].getAttribute("alt")
+
+    if(!title || title == null) return
+
+    // Reads the year value from the previewModal--detailsMetadata-left .year class
+    let yearElement = document.querySelectorAll('.previewModal--detailsMetadata-left .year')
+    let year = (yearElement === undefined  || yearElement[0] === undefined )  ? null : yearElement[0].innerText
+
+    if(!year || year == null) return
+
+    // Removes special characters from the title
+    title = title.replace(/[^a-z0-9\s]/gi, '')
+
+    // Removes spaces from the title
+    title = title.replace(/ /g, '')
+    let divId = 'ratings_' + title + '_' + year
+
+    // Fetches the element with class name present in divId
+    let ratingsElement = document.querySelectorAll(('#' + divId))
+
+    // Checks if the class exists. If not, it calls other functions
+    // which create the div and add it to the DOM
+    if(ratingsElement === undefined  || ratingsElement[0] === undefined ) {
+        //let ratings = fetchRatings(title) // Calls the backend function to fetch ratings
+        ratings = {
+            "imdb": 8.8,
+            "rotten-tomatoes": 8.7,
```

Comment:
can we use camel case rottenTomatoes?

Reply:
Also, please sync up with @lgdeloss on how backend is returning the data model. His PR is out and we are discussing these there as well